packages/diagrams/frontend/sirius-components-diagrams/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "@eclipse-sirius/sirius-components-diagrams",
+  "version": "2022.7.8",
+  "author": "Eclipse Sirius",
+  "license": "EPL-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/eclipse-sirius/sirius-components"
+  },
+  "publishConfig": {
+    "registry": "https://npm.pkg.github.com/"
+  },
+  "main": "./dist/sirius-components-diagrams.umd.js",
+  "module": "./dist/sirius-components-diagrams.es.js",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "exports": {
+    ".": {
+      "require": "./dist/sirius-components-diagrams.umd.js",
+      "import": "./dist/sirius-components-diagrams.es.js"
+    }
+  },
+  "scripts": {
+    "build": "vite build && cp src/index.d.ts dist/ && mkdir -p dist/representation && cp src/representation/DiagramRepresentation.d.ts dist/representation/",
```

Comment:
Not sure I understand this one (quite different from other packages).
Probably related to sprotty/inversify/reflect-metadata?

Reply:
I cannot generate the types since snabbdom and its JSX pragma are creating tons of issues so I've created and copied the two type declaration files manually.